activesupport/lib/active_support/dependencies/autoload.rb

```diff
@@ -69,11 +61,8 @@ def eager_autoload
     end
 
     def eager_load!
-      @_autoloads.each_value { |file| require file }
-    end
-
-    def autoloads
```

Comment:
Technically this wasn't marked as `:nodoc:` so it's supposed to go though deprecation first. I really don't see a use case for it though.